app/views/items/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for [@user, @user.items.build], remote: true do |f| %>
```

Comment:
the `remote: true` option means that this form is supposed to be able to submit asynchronously (using AJAX) instead of a page reload.  But, it doesn't look like your create action in your controller is set up to respond to AJAX.